src/simulation/LoadGenerator.cpp

```diff
@@ -969,7 +1062,7 @@ LoadGenerator::execute(TransactionFramePtr& txf, LoadGenMode mode,
     switch (mode)
     {
     case LoadGenMode::CREATE:
-        txm.mAccountCreated.Mark(batchSize);
+        txm.mAccountCreated.Mark(txf->getNumOperations());
```

Comment:
mode Pay is numOperations, is there something better than batchSize for Create? I am not following why need this change?